tasks/eth/base-003-holocene-fp-upgrade/input.json

```
@@ -0,0 +1,67 @@
+{
+  "chainId": 8453,
```

The chainId should be mainnet (`1`) imo. 
I don't know why the chainId is 8453, the task would be executed on mainnet. 
In the current script, I don't know where the `chainId` is used but for consistency with previous [task](https://github.com/ethereum-optimism/superchain-ops/pull/352/files#diff-9fe6757affc75ddec67b6b5965a760112ff204a06de6a16bce41252b1e87e61a) probably better to have 1.
```suggestion
  "chainId": 1,
```

---

Ah true, will fix. This field is meaningless anyways 😂